packages/lib/errors.ts

```diff
@@ -30,6 +30,7 @@ export async function handleErrorsJson<Type>(response: Response): Promise<Type>
 
 export function handleErrorsRaw(response: Response) {
   if (response.status === 204) {
+    console.error({ response });
```

Comment:
This can be omitted but I would like to take a deep dive to see in what scenarios we return 204 {}